ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -340,23 +341,16 @@ def ocs_install_verification(
         ), "CSI_ENABLE_SNAPSHOTTER value is not set to 'false'."
         log.info("Verified: CSI snapshotter is not present.")
 
-    # Verify pool crush rule is with "type": "zone"
+    # Verify pool crush rule is with "zone" type or "host" type
     if utils.get_az_count() == 3:
-        log.info("Verifying pool crush rule is with type: zone")
-        crush_dump = ct_pod.exec_ceph_cmd(ceph_cmd="ceph osd crush dump", format="")
-        pool_names = [
-            constants.METADATA_POOL,
-            constants.DEFAULT_BLOCKPOOL,
-            constants.DATA_POOL,
-        ]
-        crush_rules = [
-            rule for rule in crush_dump["rules"] if rule["rule_name"] in pool_names
-        ]
-        for crush_rule in crush_rules:
-            assert [
-                item for item in crush_rule["steps"] if item.get("type") == "zone"
-            ], f"{crush_rule['rule_name']} is not with type as zone"
-        log.info("Verified - pool crush rule is with type: zone")
+        pool_crush_verification(pool_type="zone")
+    elif (
+        config.DEPLOYMENT.get("local_storage")
+        and ocs_version >= 4.7
```

Comment:
The var `ocs_version` is currently the 'str' type, and we need to convert it to float. maybe we can replace the line above with this line instead: 
`and float(ocs_version) >= 4.7`